### DISCUSSION TITLE
formGrid  组件在 Modal 中会导致偶发布局失效

### DISCUSSION THREAD

**Discussion by qiphon** [alibaba/formily#3770] [Q&A 1.x]
<modal >
        <SchemaField
              schema={schemas}
            />
</Modal>

目前的解决方案是去除Modal动画

**Comment by DAOLIDEWONIU** [comment]
有更好的解决方法吗？

**Reply by qiphon** [reply]
目前没有

**Comment by millerhamburger** [comment]
增加最小列数配置，可以避免这个问题
